README.md

```diff
@@ -72,3 +72,19 @@ render((
   </Router>
 ), document.getElementById('root'))
 ```
+
+#### If you're using React Router 4's BrowserRouter you can incorporate qhistory like this:
+
+```js
+class QueryRouter extends BrowserRouter {
+  constructor(...args) {
+    super(...args)
+
+    this.history = qhistory(
+      createBrowserHistory(),
```

Comment:
`createBrowserHistory` should be receiving the props that were passed to the component (`basename`, etc.).